modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -12,25 +12,19 @@ In the table, features are marked with the following statuses:
 * *GA*: _General Availability_
 
 |===
-|*OpenShift GitOps* 7+|*Component Versions*|*OpenShift Versions*
+|*OpenShift GitOps* 8+|*Component Versions*|*OpenShift Versions*
 
-|*Version*|*kam*|*Helm*|*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex*||
-|1.6.0 |0.0.46 TP |3.8.1 GA |4.4.1 GA |2.4.5 GA |2.4.5 GA |2.30.3 GA |4.8-4.11 |GA
-|1.5.0 |0.0.42 TP |3.8.0 GA |4.4.1 GA |2.3.3 GA |0.4.1 TP |2.30.3 GA |4.8-4.11 |GA
-|1.4.0 |0.0.41 TP |3.7.1 GA |4.2.0 GA |2.2.2 GA |0.2.0 TP |2.30.0 GA |4.7-4.11 |GA
-|1.3.0|0.0.40|3.6.0|4.2.0|2.1.2|0.2.0|2.28.0|4.7-4.9|GA
-|1.2.0|0.0.38|3.5.0|3.9.4|2.0.5|0.1.0|N/A|4.8|GA
-|1.1.0|0.0.32|3.5.0|3.9.4|2.0.0|N/A|N/A|4.7|GA
+|*Version* |*kam*    |*Helm*  |*Kustomize* |*Argo CD*|*ApplicationSet* |*Dex*     |*RH SSO* |*Notifications Controller*|
+|1.7.0    |0.0.46 TP |3.10.0 GA|4.5.7 GA   |2.5.4 GA |2.4.5 GA       |2.35.1 GA |7.5.1 GA |2.4.5 TP|4.8-4.11
+|1.6.0    |0.0.46 TP |3.8.1 GA|4.4.1 GA   |2.4.5 GA |2.4.5 GA       |2.30.3 GA |7.5.1 GA |2.4.5 TP|4.8-4.10
+|1.5.0    |0.0.42 TP|3.8.0 GA|4.4.1 GA   |2.3.3 GA |0.4.1 TP        |2.30.3 GA |7.5.1 GA ||4.8-4.10
+|1.4.0    |0.0.41 TP|3.7.1 GA|4.2.0 GA   |2.2.2 GA |0.2.0 TP        |2.30.0 GA |7.4.0 GA ||4.7-4.9
+|1.3.0    |0.0.40 TP|3.6.0 GA|4.2.0 GA   |2.1.2 GA |0.2.0 TP        |2.28.0 GA |7.4.0 GA ||4.7-4.9
 
 |===
 
-[NOTE]
-====
-To use {gitops-title} v1.3 on {product-title} 4.6, do a fresh installation of {gitops-title}. There is no upgrade path from {gitops-title} 1.0 (TP) on OpenShift 4.6.
-====
```

Comment:
@shipsing **Observation:**  [GITOPS-1756](https://issues.redhat.com/browse/GITOPS-1756) is already documented in 4.8 OCP under Fixed Issues. 

**Suggestion:**

- So no need of repeating it in RN 1.4.2 and 1.3.3. 

- Also, add the RN module 1.2.2 in OCP version 4.9 and later.

Reply:
new issue created to track this comment: https://issues.redhat.com/browse/RHDEVDOCS-4958

Reply:
Resolved in this PR: https://github.com/openshift/openshift-docs/pull/55464#discussion_r1100082257